charts/openbao/templates/server-statefulset.yaml

```
@@ -20,7 +20,7 @@ metadata:
   {{- template "openbao.statefulSet.annotations" . }}
 spec:
   serviceName: {{ template "openbao.fullname" . }}-internal
-  podManagementPolicy: Parallel
+  podManagementPolicy: {{ .Values.server.podManagementPolicy | default "Parallel" }}
```

```suggestion
  podManagementPolicy: {{ .Values.server.podManagementPolicy }}
```

The default doesn't need to be mentioned here as it's already part of the default `values.yaml`